PWGEM/PhotonMeson/TableProducer/skimmerGammaCalo.cxx

```diff
@@ -63,30 +67,41 @@ struct skimmerGammaCalo {
     LOG(info) << "| M02 cut: " << minM02 << " < M02 < " << maxM02 << std::endl;
   }
 
-  void processRec(aod::Collision const&, aod::EMCALClusters const& emcclusters, aod::EMCALClusterCells const& emcclustercells, aod::EMCALMatchedTracks const& emcmatchedtracks, aod::Tracks const& tracks)
+  void processRec(aod::Collision const&, aod::EMCALClusters const& emcclusters, aod::EMCALClusterCells const& emcclustercells, aod::EMCALMatchedTracks const& emcmatchedtracks, aod::FullTracks const& tracks)
   {
     for (const auto& emccluster : emcclusters) {
       historeg.fill(HIST("hCaloClusterEIn"), emccluster.energy());
       historeg.fill(HIST("hCaloClusterFilter"), 0);
 
+      // timing cut
       if (emccluster.time() > maxTime || emccluster.time() < minTime) {
         historeg.fill(HIST("hCaloClusterFilter"), 1);
         continue;
       }
+      // M02 cut
       if (emccluster.m02() > maxM02 || emccluster.m02() < minM02) {
         historeg.fill(HIST("hCaloClusterFilter"), 2);
         continue;
       }
 
+      // Skimmed cell table
       auto groupedCells = emcclustercells.sliceBy(CellperCluster, emccluster.globalIndex());
```

Comment:
That would be nice, can you check? I was told to use sliceBy

Reply:
I just checked, Anton once suggested to me to use sliceByCached instead of sliceBy. However I think this is only faster if some requirements are met. If you want search the O2 Analysis channel for sliceByCached to learn more but I think this is sth that can be done later. Just wanted to mention it so maybe we don't forget it

Reply:
I talked with Anton and rule of thumb: Preslice + slicyBy is basically like slicyByCached and sliceByCached is only needed when using Partitions.

Reply:
awright, thanks for finding this out!